### PR TITLE
Support Github Enterprise

### PR DESCRIPTION
**Pull request by chitran96**
Implement https://github.com/ewjoachim/coverage-comment-action/issues/65

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -6,6 +6,11 @@ description: >
   Publish diff coverage report as PR comment, and create a coverage badge
   to display on the readme.
 inputs:
+  GITHUB_URL:
+    description: >
+      A Github base url, api.github.com by default but can be different in GitHub Enterprise
+    default: "https://api.github.com"
+    required: false
```
Comment (ewjoachim) [inline action.yml]: Actually, it's already injected by GHA as `GITHUB_API_URL`. You don't need to add it in the inputs.
In settings.py, you can read it from the environment directly.

(Also, please make sure you wanted to contribute here and not to https://github.com/py-cov-action/python-coverage-comment-action, people have made the mistake in the past)

Comment (chitran96) [reply action.yml]: great! I didn't know that. And yes, it's my intention to contribute to this action, our project written in Scala.

**Review by ewjoachim — APPROVED** [review]

**Comment by ewjoachim** [comment]
Thank you @chitran96 !

As you may have seen from the Readme, this action is not particularly maintained, but if you're interested, I'm happy to share / handover :)

**Comment by chitran96** [comment]
yeah since we're going to use this action in our project, I'm open to share with you :D

**Pull request merged**